searx/templates/simple/macros.html

```diff
@@ -25,8 +25,7 @@
   <span class="url_o{{loop.index}}"><span class="url_i{{loop.index}}">{{- part -}}</span></span>
   {%- endfor %}
   {{- result_close_link() -}}
-  {%- if result.img_src %}{{ result_open_link(result.url) }}<img class="image" src="{{ image_proxify(result.img_src) }}" title="{{ result.title|striptags }}" loading="lazy" width="200" height="200">{{ result_close_link() }}{% endif -%}
-  {%- if result.thumbnail %}{{ result_open_link(result.url) }}<img class="thumbnail" src="{{ image_proxify(result.thumbnail) }}" title="{{ result.title|striptags }}" loading="lazy" width="200" height="200">{{ result_close_link() }}{% endif -%}
+  {%- if result.thumbnail %}{{ result_open_link(result.url) }}<img class="thumbnail" src="{{ image_proxify(result.thumbnail) }}" title="{{ result.title|striptags }}" loading="lazy">{{ result_close_link() }}{% endif -%}
```

Comment:
> it surprises me that an unknown height helps the lazy loading

We had some fight with lazy load in the past (2021) --> https://github.com/searxng/searxng/pull/573

As far as I can remember it was a problem when height was unclear and the browser needed a value to calculate the layout .. may be browser have changed since .. in this PR I removed the fixed size of 200x200 in the template.

----

The details seem to be very complex and I don't know if the browser behavior has changed in the meantime:

To be sure, I just tested it again: in the current Firefox and Chrome the lazy load works, both for the results from the `image.html` and for those from the `default.html` template